pybamm/solvers/c_solvers/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = []
```

Comment:
This one is mentioned by CodeCov as a line we have missing coverage for, so I think we can remove the line.

```suggestion
```